helm-charts/dnsbl-exporter/values.yaml

```diff
@@ -0,0 +1,339 @@
+  periodSeconds: 5
+  timeoutSeconds: 3
+  failureThreshold: 6
+  successThreshold: 1
+## @param customLivenessProbe Custom livenessProbe that overrides the default one
+##
+customLivenessProbe: {}
+## @param customReadinessProbe Custom readinessProbe that overrides the default one
+#
+customReadinessProbe: {}
+## Custom startupProbe block
+customStartupProbe: {}
+
+image:
+  registry: ghcr.io
+  repository: luzilla/dnsbl_exporter
+  tag: v0.6.0
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+
+configuration:
+  rbls:
+    # 0SPAM
+    - bl.0spam.org
+    - rbl.0spam.org
+    # NIXSPAM
+    - ix.dnsbl.manitu.net
+    # Spamhaus Zen
+    - zen.spamhaus.org
+  targets:
+    - smtp.fastmail.com
+  resolver: "1.1.1.1:53"
+
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # Labels to add to the service account
+  labels: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+## Application Pod Disruption Budget configuration
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+## @param pdb.create Enable a Pod Disruption Budget creation
+## @param pdb.minAvailable Minimum number/percentage of pods that should remain scheduled
+## @param pdb.maxUnavailable Maximum number/percentage of pods that may be made unavailable
+##
+pdb:
+  create: false
+  minAvailable: 1
+  maxUnavailable: ""
+## Configure Pods Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+## @param podSecurityContext.enabled Enabled Ghost pods' Security Context
+## @param podSecurityContext.fsGroup Set Ghost pod's Security Context fsGroup
+##
+podSecurityContext:
+  enabled: true
+  # Volumes that support ownership management are modified to be owned and writable by the GID specified in fsGroup
+  fsGroup: 101
+## Configure Container Security Context (only main container)
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+## @param containerSecurityContext.enabled Enabled Ghost containers' Security Context
+## @param containerSecurityContext.runAsUser Set Ghost container's Security Context runAsUser
+## @param containerSecurityContext.runAsNonRoot Set Ghost container's Security Context runAsNonRoot
+##
+containerSecurityContext:
+  enabled: true
+  runAsUser: 100
+  runAsNonRoot: true
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
```

Comment:
Maybe I am confusing `containerSecurityContext` with `securityContext`.

Reply:
This is about Linux capabilities. Check **[man 7 capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html)**. In short - allows to cut some permissions for root, or add some extra for non-privileged users (like open port <1024).